docs/UserGuide.md

```diff
@@ -156,6 +156,14 @@ Before continuing, here are some important information you need to know about th
 
 Before diving into our features, do note that we set some specifications for naming and such. You might want to read this section if you keep getting an `Invalid command format!` message, or want to find out more intricate details about InternBuddy. Otherwise, skip to [add command](#adding-a-company-add) to begin.
 
+<box type="warning" seamless>
+
+**Note about extra parameters:** We advise that you follow the command format of each command strictly (some exceptions apply) to avoid
+unexpected behaviour when running commands. This means you **should not** put extra parameters unless allowed, and you **should
+not** put parameters that are not recognised by the command.
```

Comment:
Phrasing suggestion
```diff
- **Note about extra parameters:** We advise that you follow the command format of each command strictly (some exceptions apply) to avoid unexpected behaviour when running commands. This means you **should not** put extra parameters unless allowed, and you **should not** put parameters that are not recognised by the command.
+ **Note about extra parameters:** We advise that you follow the command format of each command **strictly** (some exceptions apply) to avoid any unexpected behaviour when running our specified commands. This means that you **should not** put any extra parameters unless mentioned, and you **should not** put any parameters or prefixes that are not recognised by the command.
```